From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: gellert <gellert.hegyi@around.co>
Date: Fri, 13 Aug 2021 11:23:10 +0200
Subject: linux window transparency fix

This patch allows the frameless window to be transparent on windows.

diff --git a/ui/base/x/x11_util.cc b/ui/base/x/x11_util.cc
index a17c1603e8af4083cbaef33937f7a6ccaddc2284..7e8c775952690a7ad9a3a41bfa9bc8ff3327db60 100644
--- a/ui/base/x/x11_util.cc
+++ b/ui/base/x/x11_util.cc
@@ -1085,6 +1085,10 @@ void XVisualManager::ChooseVisualForWindow(bool want_argb_visual,
   bool use_argb = want_argb_visual && ArgbVisualAvailable();
   x11::VisualId visual = use_argb ? transparent_visual_id_ : opaque_visual_id_;
 
+#if defined(OS_LINUX)
+  use_argb = true;
+#endif
+
   if (visual_id)
     *visual_id = visual;
   bool success = GetVisualInfo(visual, depth, colormap, visual_has_alpha);
